src/table/components/virtualized-table/Wrapper.tsx

```diff
@@ -1,21 +1,26 @@
-import React, { useState } from 'react';
+import React, { useMemo, useState } from 'react';
 import { WrapperProps } from './types';
 import PaginationContent from '../PaginationContent';
 import { StyledTableWrapper } from '../../styles';
 import { PageInfo, TableData } from '../../../types';
 import FooterWrapper from '../FooterWrapper';
-import TableContainer from './TableContainer';
+import Table from './TableContainer';
```

Comment:
how come the rename? I think it is good to keep the name of the component and the file the same

Reply:
Good question. Got lint issues when the memoized TableContainer and TableContainer was both exported with the same name, but the memoized version as default. Did not look further into how to best solve it. But I do agree that I would prefer the same name.